### ISSUE TITLE
Create dbt package generator using GenAI and PyAirbyte

### ISSUE THREAD

**Issue by aaronsteers** [airbytehq/PyAirbyte-Hackathon#6]

**Comment by Hashcode-Ankit** [comment]
hi @aaronsteers  can I have more information on this topic would love to contribute to it.

**Comment by siddhant3030** [comment]
can you give us the summary on this?

**Comment by marcosmarxm** [comment]
@Hashcode-Ankit it is yours

**Comment by aaronsteers** [comment]
@Hashcode-Ankit - I've updated the above with a description. Admittedly, this is a large and ambitious project. Let us know if it is still interesting, and/or if you have any questions or changes you would like to propose.

**Comment by Hashcode-Ankit** [comment]
@aaronsteers Yes as I mentioned on Slack as well, I am very much excited about it, and it matches my previous work on dbt and dbt-clickhouse.

**Comment by Hashcode-Ankit** [comment]
Hi @aaronsteers @marcosmarxm  as discussed on slack i have pushed the code for it : https://github.com/Hashcode-Ankit/pyairbyte-dbt
Need some more guidance on task to complete the exact use case @aaronsteers let me know if we can connect.
